### ISSUE TITLE
undefined method `each' for nil:NilClass

### ISSUE THREAD

**Issue by sega-yarkin** [SumoLogic/fluentd-kubernetes-sumologic#87]
Hi, I have an error in logs with the latest version:
```
#<NoMethodError: undefined method `each' for nil:NilClass>
/fluentd/plugins/filter_kubernetes_sumologic.rb:84:in `filter'
```

There is no `labels` field in `kubernetes` hash in my case.
Thank you!

**Comment by frankreno** [comment]
ahh thanks for the issue, I will look into this soon.

**Comment by frankreno** [comment]
This has been fixed in v2.0.0
